lib/decrypt.rb

```diff
@@ -6,11 +6,11 @@
 
 handle = File.open(ARGV[0], "r")
 
-message = handle.read
+ciphertext = handle.read
```

Comment:
WHAT A GREAT CHANGE DAVID
TOTALLY WORTH THE HEADACHE DAVID
IT'S OKAY DAVID I DIDN'T CARE ABOUT FIGURING OUT CRACK ANYWAY DAVID
THANKS